package.json

```
@@ -75,6 +64,17 @@
 	},
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "0.7.0",
+		"@types/is-ci": "^2.0.0",
+		"@types/lodash.throttle": "^4.1.6",
+		"@types/node": "*",
```

```suggestion
		"@types/node": "^10.17.26",
```

Ideally this should be set to the lowest major supported by your `engines` field :)